src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -41,7 +46,111 @@ public void InterpreterTestCase(ExpressionTestCase testCase)
                     Skip.If(true, prefix + msg);
                     break;
             }
-        } 
+        }        
+
+        [InterpreterTheory(Skip="NotReadyYet")]
+        [TxtFileData("ExpressionTestCases\\NotYetReady", "InterpreterExpressionTestCases", nameof(InterpreterRunner))]
+        public void InterpreterTestCase_NotReadyTests(ExpressionTestCase testCase)
+        {
+            // This is running against embedded resources, so if you're updating the .txt files,
+            // make sure they build is actually copying them over. 
+            Assert.True(testCase.FailMessage == null, testCase.FailMessage);
+
+            _runner = new InterpreterRunner();
+            var (result, msg) = _runner.RunAsync(testCase).Result;
+            var prefix = $"Test {Path.GetFileName(testCase.SourceFile)}:{testCase.SourceLine}: ";            
+
+            switch (result)
+            {
+                case TestResult.Pass:
+                    break;
+
+                case TestResult.Fail:
+                    Stats.AddOrUpdate(prefix, (testCase, msg), (x, z) => throw new Exception("Oups"));
+                    Assert.True(false, prefix + msg);
+                    break;
+
+                case TestResult.Skip:
+                    Skip.If(true, prefix + msg);
+                    break;
+            }
+        }
+
+        private enum TestCategory
+        {
+            DidNotReturnAValue, // contains "did not return a value"
+            WrongErrorKind, // Received an error, but expected kind=InvalidArgument and received BadLanguageCode
+            ErrorValue, // contains Microsoft.PowerFx.Core.Public.Values.ErrorValue
+            InvalidResult, //  Expected: 31. actual: 12
+            ThrewException, // contains "Threw exception"
+            Unknown,
+        }
+
+        private TestCategory GetTestCategory(string str)
+        {
+            if (str.Contains("did not return a value"))
+            {
+                return TestCategory.DidNotReturnAValue;
+            }
+            else if (new Regex(@"Received an error, but expected kind=[a-zA-Z0-9]* and received").IsMatch(str))
+            {
+                return TestCategory.WrongErrorKind;
+            }
+            else if (str.Contains("Microsoft.PowerFx.Core.Public.Values.ErrorValue"))
+            {
+                return TestCategory.ErrorValue;
+            }
+            else if (str.Contains("Expected: ") && str.Contains(". actual: "))
+            {
+                return TestCategory.InvalidResult;
+            }
+            else if (str.Contains("Threw exception"))
+            {
+                return TestCategory.ThrewException;
+            }
+
+            return TestCategory.Unknown;            
+        }
+
+        private static readonly ConcurrentDictionary<string, (TestCase TestCase, string Message)> Stats = new ();
+
+        [Fact]
+        public void GetStats()
+        {
+            Thread.Sleep(1000); // Let's things start
```

Comment:
With Thread.Sleep, Let's not check this in :)

Reply:
WIP code, just temporary to extract some statistics
I won't check that in.

Reply:
Removed